packages/eslint-plugin/src/rules/class-literal-property-style.ts

```diff
@@ -45,10 +49,13 @@ export default util.createRule<Options, MessageIds>({
         'Enforce that literals on classes are exposed in a consistent style',
       recommended: 'strict',
     },
-    fixable: 'code',
+    hasSuggestions: true,
     messages: {
       preferFieldStyle: 'Literals should be exposed using readonly fields.',
+      preferFieldStyleSuggestion:
+        'Try replacing the literals with readonly fields.',
```

Comment:
I wasn't sure what messaging to use here, I'm open for other suggestions 🙂

Reply:
Ah yeah these can be tricky. Searching around other rules that have `hasSuggestions: true`, we normally go with language like _"Do X with Y"_. Here I think that'd be:

```diff
- 'Try replacing the literals with readonly fields.'
+ 'Replace the literals with readonly fields.'
```

What do you think about that?

Reply:
Yes, that sounds good, I've updated both suggestion messages.